drivers/pulse_counter/include/pulse_counter_params.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2017 UC Berkeley
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_pulse_counter
+ *
+ * @{
+ * @file
+ * @brief       Default configuration for PULSE_COUNTER devices
+ *
+ * @author      Hyung-Sin Kim <hs.kim@cs.berkeley.edu>
+ */
+
+#ifndef PULSE_COUNTER_PARAMS_H
+#define PULSE_COUNTER_PARAMS_H
+
+#include "board.h"
+#include "pulse_counter.h"
+#include "saul_reg.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef PULSE_COUNTER_GPIO
+#define PULSE_COUNTER_GPIO GPIO_PIN(PA,18)
+#endif
+
+#ifndef PULSE_COUNTER_GPIO_FLANK
+#define PULSE_COUNTER_GPIO_FLANK GPIO_FALLING
+#endif
+
+#define PULSE_COUNTER_PARAMS       { .gpio = PULSE_COUNTER_GPIO, \
```

Comment:
You can wrap this with an `#ifndef PULSE_COUNTER_PARAMS .. #endif`

Reply:
addressed